installing/installing_oci/installing-oci-assisted-installer.adoc

```diff
@@ -51,7 +51,9 @@ include::modules/installing-troubleshooting-assisted-installer-oci.adoc[leveloff
 [role="_additional-resources"]
 .Additional resources
 
+* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting {product-title} on {oci} (Oracle documentation)]
```

Comment:
Further down, you use _in the Oracle documentation_. I think that phrasing is more clear. Either way, I think we should be consistent. 

```suggestion
* link:https://docs.oracle.com/iaas/Content/openshift-on-oci/openshift-troubleshooting.htm[Troubleshooting {product-title} on {oci}] in the Oracle documentation
```

Reply:
OK as per recent [guidelines ](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#lead-in-link-sentence-style-for-the-link-attribute) updates.